packages/boolean/src/index.ts

```diff
@@ -7,8 +7,8 @@ const reducers = {
     off() {
         return false;
     },
-    toggle(value: boolean) {
-        return !value;
+    toggle(prevState: boolean, arg: any) {
```

Comment:
`arg` should be boolean, strong typed hooks are always better

Reply:
The type will be incorrect when i use the function as a event handler or some other callback，e.g.
```
<div on-click = { useToglle }></on-click>
<custom-comp on-selected= { useToggle }></custom-comp>
```

Reply:
👌